File: B/C++/Product.cpp
#include <iostream>
#include <string>
using namespace std;

class Product{
    private:
        string id;
        int price;
    public:
        Product(){
            this->id="";
            this->price=0;
        }

        void setID(string i){
            this->id = i;
        }

        string getID(){
            return this->id;
        }

        void setPrice(int p){
            this->price = p;
        }

        int getPrice(){
            return this->price;
        }
};

File: B/PHP/Product.php
<?php

class Product{
    private $id = 0;
    private $price = 0;

    public function __construct($id = 0, $price = 0){
        $this->id = $id;
        $this->price = $price;
    }

    public function setID($id){
        $this->id = $id;
    }

    public function getID(){
        return $this->id;
    }

    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }
}

?>